packages/atomic/src/components/search/facets/atomic-facet/atomic-facet.tsx

```diff
@@ -128,11 +136,38 @@ export class AtomicFacet implements InitializableComponent {
    * The field whose values you want to display in the facet.
    */
   @Prop({reflect: true}) public field!: string;
+  /**
+   * The tabs on which the facet can be displayed. This property complements `tabs-excluded`.
+   *
+   * Set this property as a stringified JSON array, e.g.,
+   * ```html
+   *  <atomic-facet tabs-included='["tabIDA", "tabIDB"]'></atomic-facet>
+   * ```
+   * If you don't set this property, or set it to `'[]'`, the facet can be displayed on any tab. Otherwise, the facet can only be displayed on the specified tabs. In either case, the facet won't be displayed on any of the tabs specified in the `tabs-excluded` property (exclusion takes precedence).
+   */
+  @ArrayProp()
+  @Prop({reflect: true, mutable: true})
+  public tabsIncluded: string[] | string = '[]';
```

Comment:
What's the reasoning behind  _If you don't set this property, or set it to `'[]'`_ ? Why would they prefer to write `[]` instead of nothing ?

Why can't it just be an array of string with an empty default value ?
```
  @ArrayProp()
  @Prop({reflect: true, mutable: true})
  public tabsIncluded: string[] = [];
```

Reply:
The idea was to allow providing just a tab name when a single tab is included/excluded.